tool/tbot/main.go

```diff
@@ -123,6 +123,10 @@ func Run(args []string, stdout io.Writer) error {
 			return onKeypairCreateCommand(ctx, globalCfg, keypairCreateCmd)
 		}),
 
+		cli.NewCopyBinariesCommand(app, func(cbc *cli.CopyBinariesCommand) error {
```

Comment:
I think you've forgotten to add the file in lib/tbot/cli that defines this 😓

Reply:
🤦 my enormous list of untracked files finally comes back to bite me, I forgot to commit it. Fixed now!